### PR TITLE
Fix for scroll issue in the table resize plugin

### PR DESCRIPTION
**Pull request by SimonW97**
If the editor is in a scrollable container and you start scrolling the TableResize plugin breaks.

The error can be reproduced by adding min-height to the .mainPane container on the demo page and setting it to a value at which the scrollbar appears. If you then add a table and scroll down a bit the resize function is broken.
The reason seems to be that in the onMouseMove event the cursor coordinates are directly compared with the table rect coordinates. But the cursor coordinates are the absolute position on the page and the table rect coordinates are relative to the browser window. So subtracting the scroll position from the cursor coordinates should fix the issue.

I was able to reproduce the issue and also test the fix in Firefox, Chrome und Edge.

### CODE REVIEW

File: packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts [hunk 1]
```diff
@@ -61,7 +61,8 @@ export default class TableResize implements EditorPlugin {
         }
 
         this.ensureTableRects();
-        const { pageX: x, pageY: y } = e;
+        const x = e.pageX - window.scrollX;
```
Comment (JiuqingSong) [inline packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts:64]: This is a good finding. I'm afraid each place where e.pageX/Y is used will have the same issue.

And here instead of using the global window, we need to retrieve window object from editor instead, because the global window may not be the window that editor is in.

Comment (SimonW97) [reply packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts:64]: Ok, but then I'm afraid I don't know the code well enough to fix this in all the necessary places.

Comment (JiuqingSong) [reply packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts:64]: We can fix other place separately. Please just change the global window here to:
```ts
  this.editor.getDocument().defaultView
```

Thanks.

**Review by JiuqingSong — APPROVED** [review]
